### DISCUSSION TITLE
Moving ICESat/ICESat-2/OIB scripts to separate repositories

### DISCUSSION THREAD

**Discussion by tsutterley** [pyTMD/pyTMD#72] [General]
Could make the pyTMD repository skinnier by dropping some dependencies.  Not sure if anyone else is using those programs.  Suggestions?

**Comment by mrsiegfried** [comment]
I actually really like having these in there. But I'm totally biased. They maybe do belong elsewhere in a more altimetry focused repo... 🤔

**Comment by lyingTree** [comment]
I think you are right. Those scripts should belong to another repository
Because they are not related to the purpose of this repository.

**Comment by tsutterley** [comment]
Okay, I'm making this shift. IS/IS2/OIB codes will be a separate repo called [`Grounding-Zones`](https://github.com/tsutterley/Grounding-Zones).
